### ISSUE TITLE
meadowrun-manage-ec2: command not found

### ISSUE THREAD

**Issue by weisisheng** [meadowdata/meadowrun#81]
Have tried installing in base virtual env using poetry and pip.  Can't seem to be found.  Appreciate any tips. Thanks in advance.

Ubuntu 22.04, python 3.7.11, meadowrun 0.11 // with non-sudo and sudo

**Comment by kurtschelfthout** [comment]
Thanks for trying meadowrun! I suspect it's the python 3.7 that is biting you, I recently added it to our regular test matrix and found a few issues in the process - #77 has not been released yet.

Could you try

```python
python -c "import meadowrun.manage; meadowrun.manage.main_ec2()" --help / install / ...
```
to check? If it gives some typing related import errors that would be a sure indication that is the issue.

Or try with a more recent python version if you can.

We will release an updated version with the 3.7 fixes.

**Comment by hrichardlee** [comment]
We did have a fix for a python 3.7 issue that I just released (so if you get 0.1.12 you shouldn't run into that), but that was an issue with `from typing import Literal` issue, which is not what you were running into.

My guess is that you're somehow using a pip that's related to a python version that's not on your path. We would recommend using the advice here: https://stackoverflow.com/questions/2812520/dealing-with-multiple-python-versions-and-pip which is to always call pip via `python -m pip ...` rather than just `pip` by itself. As long as the `python` you use in that command is on your path, then the `meadowrun-manage-ec2` command should work. But in general the recommendation would be to use a virtualenv. I just ran this on Ubuntu and it worked:

```shell
python3.7 -m venv py37venv
source py37venv/bin/activate
pip install meadowrun
meadowrun-manage-ec2 install
```
